src/quest_manager/templates/quest_manager/category_list.html

```diff
@@ -1,4 +1,4 @@
-{% extends "quest_manager/base.html" %}
+{% extends "courses/base.html" %}
```

Comment:
Why this change? Campaign/categories are in the quest_manager app (models/views, etc)

Reply:
the change was originally intended to keep consistent with the top right icon on the campaigns page and the icon used to represent it in the admin menu, but there is an inconsistency with site logic. change reverted.
![image](https://user-images.githubusercontent.com/105619909/183224150-81e64276-b7ea-46bf-a082-18fcbf67a74c.png)